test/util.spec.js

```diff
@@ -45,21 +45,39 @@ describe('util', () => {
   })
 
   it('.serialize and .deserialize large objects', (done) => {
-    const size = 65536 * 2
+    // larger than the default borc heap size, should auto-grow the heap
+    const size = 1 << 17 // 128Kb
     const largeObj = { someKey: [].slice.call(new Uint8Array(size)) }
-    // Configure decoder with custom size
-    dagCBOR.util.configureDecoder({ size: size + 1 })
 
     dagCBOR.util.serialize(largeObj, (err, serialized) => {
       expect(err).to.not.exist()
-      expect(Buffer.isBuffer(serialized)).to.equal(true)
+      expect(Buffer.isBuffer(serialized)).to.be.true()
 
       dagCBOR.util.deserialize(serialized, (err, deserialized) => {
         expect(err).to.not.exist()
         expect(largeObj).to.eql(deserialized)
+        // reset decoder to default
+        dagCBOR.util.configureDecoder({ size: 1 << 16, maxSize: 1 << 26 })
+        done()
+      })
+    })
+  })
 
-        // Reset decoder back to default
-        dagCBOR.util.configureDecoder()
+  it('.deserialize fail on large objects beyond maxSize', (done) => {
+    // larger than the default borc heap size, should bust the heap if we turn off auto-grow
+    const size = 1 << 17 + 1 // 124Kb + 1
```

Comment:
Nitpick: As these tests are all about sizes I would find `dataSize` clearer than just `size`. Again something you don't need to change, I just enjoy having those comments when I get reviews as naming things has lots to do with experience.